Python/Movie-Info-Telegram-Bot/bot.py

```diff
@@ -0,0 +1,165 @@
+import logging
+import requests
+import re
+import urllib.request
+import urllib.parse
+import urllib.error
+from bs4 import BeautifulSoup
+import ssl
+import itertools
+from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
+
+# Enable logging
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                    level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+TOKEN = 'YOUR TOKEN'
```

Comment:
Kindly go through the contribution guidelines, set  up an `env` file, for the Token, upload an `env.example`  file.

The steps have been mentioned in the Guidelines. In case you are having problems, this [script](https://github.com/HarshCasper/Rotten-Scripts/tree/master/Python/Github_Traffic) can help.